.changeset/dull-colts-fetch.md

```diff
@@ -0,0 +1,137 @@
+---
+'sku': major
+---
+
+Remove Storybook-related features from `sku`
+
+**BREAKING CHANGE**
+
+All Storybook-related features have been removed from `sku`. Sku no longer installs Storybook dependencies, configures Storybook, provides Storybook CLI commands, or re-exports Storybook APIs. If you are using Storybook, you will need to install Storybook dependencies yourself. See [`sku`'s Storybook documentation][storybook docs] for more information.
+
+[storybook docs]: https://seek-oss.github.io/sku/#/./docs/storybook
+
+**MIGRATION GUIDE**:
+
+#### Update your `sku.config.js`
+
+The following `sku` configuration options have been removed:
+
+- `storybookAddons`
+- `storybookPort`
+- `storybookStoryStore`
+- `storybookTarget`
+
+Please remove them from your sku configuration file.
+
+Save the value of `storybookTarget` to use later in the migration.
+
+#### Update your ignore files
+
+Sku no longer manages certain storybook related ignorefile entries.
```

Comment:
We did some googling and it seems like `ignore files` is used more often than `ignorefiles`.

```suggestion
Sku no longer manages certain Storybook-related ignore file entries.
```